apps/backend/file-service/src/test/resources/application-test.yml

```
@@ -0,0 +1,18 @@
+  jpa:
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.PostgreSQLDialect
+    defer-datasource-initialization: true
+    database-platform: org.hibernate.dialect.PostgreSQLDialect
+    show-sql: true
+    hibernate:
+      ddl-auto: update
+  sql:
+    init:
+      mode: always
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)